internal/pkg/template/workload.go

```diff
@@ -214,6 +214,7 @@ type NetworkLoadBalancerListener struct {
 	TargetPort      string
 	SSLPolicy       *string
 	Aliases         []string
+	Stickness       *bool
```

Comment:
Do we expose this in our manifest?

Reply:
This is a temporary relief for the sake of rendering the template without error. It was okay before because http couldn't be disabled and it was using `!Ref StickIness` which was actually the http stickIness. Now with http disabled, `!Ref StickIness` produces error because `Stickiness` as a parameter is taken out of the template. To avoid erroring out, I added this temporary relief.

The actual NLB health check is coming in the next PR.